lacework/resource_lacework_managed_policies.go

```diff
@@ -0,0 +1,135 @@
+package lacework
+
+import (
+	"fmt"
+	"log"
+	"strings"
+	"time"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/lacework/go-sdk/api"
+)
+
+func resourceLaceworkManagedPolicies() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceLaceworkManagedPoliciesCreate,
+		Update: resourceLaceworkManagedPoliciesUpdate,
+		Delete: resourceLaceworkManagedPoliciesDelete,
+		Read:   resourceLaceworkManagedPoliciesRead,
+
+		Importer: &schema.ResourceImporter{
+			State: importLaceworkManagedPolicies,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"policy": {
+				Type: schema.TypeSet,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"id": {
+							Type:        schema.TypeString,
+							Description: "The id of the policy",
+							Required:    true,
+						},
+						"enabled": {
+							Type:        schema.TypeBool,
+							Required:    true,
+							Description: "The state of the policy",
+						},
+						"severity": {
+							Type:     schema.TypeString,
+							Optional: true,
+							Description: "The severity for the policy. Valid severities are: " +
+								"Critical, High, Medium, Low, Info",
+							StateFunc: func(val interface{}) string {
+								return strings.TrimSpace(strings.ToLower(val.(string)))
+							},
+							ValidateDiagFunc: ValidSeverity(),
+						},
+					},
+				},
+				Required:    true,
+				Description: "A list of Lacework managed policies",
+			},
+		},
+	}
+}
+
+func resourceLaceworkManagedPoliciesCreate(d *schema.ResourceData, meta interface{}) error {
+	d.SetId(time.Now().UTC().String())
+	return resourceLaceworkManagedPoliciesUpdate(d, meta)
+}
+
+func resourceLaceworkManagedPoliciesUpdate(d *schema.ResourceData, meta interface{}) error {
+	lacework := meta.(*api.Client)
+	policies, err := getBulkUpdatePolicies(d, meta)
+
+	if err != nil {
+		return err
+	}
+
+	log.Printf("[INFO] Updating Policies with data:\n%+v\n", policies)
+	_, updateErr := lacework.V2.Policy.UpdateMany(policies)
+	if updateErr != nil {
+		return updateErr
+	}
+	log.Printf("[INFO] Updated Policies with data:\n%+v\n", policies)
+	return nil
+}
+
+func resourceLaceworkManagedPoliciesRead(d *schema.ResourceData, meta interface{}) error {
+	return nil
```

Comment:
I think the read operation is really important for both, detect changes and for importing the current state of the managed policies.

Reply:
Implemented the read operation and verified that the manual change can be detected. Added screenshots in summary.